src/Pages/Home.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import NavBar from '../component/Navbar';
+import '../component/styles/pages.css';
+
+const Home = () => (
+  <>
+    <NavBar />
```

Comment:
- [ ] kindly remove the `navbar` in this component and import it in the app.js to call it for once instead of calling it in each component